components/catalogs/back/src/Application/Persistence/User/UserNotFoundException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akeneo\Catalogs\Application\Persistence\User;
```

Comment:
```suggestion
namespace Akeneo\Catalogs\Application\Exception\User;
//or
namespace Akeneo\Catalogs\Application\Exception;
```
As it was done for CatalogNotFoundException